packages/servers/express-helloworld/index.js

```diff
@@ -0,0 +1,15 @@
+const start = process.hrtime();
+const express = require('express');
+
+const app = express();
+app.get('*path', (req, res) => {
+  res.status(200).json({
+    hello: 'world!',
+    url: req.url,
+    headers: req.headers
+  });
+});
+
+app.listen(process.env.PORT || 3000, () => {
+  console.log(`startup: ${process.hrtime(start)}`);
```

Comment:
`process.hrtime` is legacy now. I need to get in the habit of using `process.hrtime.bigint` now. Will update before we merge.